.eslintrc.js

```diff
@@ -1,20 +1,43 @@
 module.exports = {
+  env: {
+    browser: true,
+    es6: true,
+  },
+  extends: [
+    'airbnb',
+    'prettier',
+  ],
+  globals: {
+    Atomics: 'readonly',
+    SharedArrayBuffer: 'readonly',
+  },
   parser: '@typescript-eslint/parser',
   parserOptions: {
-    project: './tsconfig.json',
```

Comment:
We need this to let know eslint typescript parser how to use rules that depends on types: this is different from eslint for javascript where there are no rules that depends on types.

Reply:
I will add it back 👍

Reply:
This has been added back.